src/Containers/TopLandingSection/index.scss

```diff
@@ -0,0 +1,18 @@
+.top-landing-section{
+    &__container{
+        &--title{
+            font-size: 3vh;
+            margin-bottom: 8vh;
+        }
+        &--slogan{
+            font-size: 4vh;
+            margin-bottom: 20vh;
+            padding-right: 4vw;
+        }
+        &--description{
+            font-size: 3vh;
+            margin-bottom: 5vh;
+            padding-right: 4vw;
+        }
+    }
+}
```

Comment:
An empty ine shouls be added the end of each files to smothly be treated in different platforms or OS

Reply:
Cool, but there is a task to help fix these types of errors.